game/game/api/shortcuts.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+import game.constants
+from common.ctype import ctype
+from common.api_handlers import l2_request_handler
+from common.template import Template, Parameter
+
+LOG = logging.getLogger(f"l2py.{__name__}")
+
+
+@l2_request_handler(game.constants.GAME_REQUEST_SHORTCUT_REG,
+                    Template([
+                        Parameter(id="type", start=0, length=4, type=ctype.int32),
+                        Parameter(id="slot", start="$type.stop", length=4, type=ctype.int32),
+                        Parameter(id="id", start="$slot.stop", length=4, type=ctype.int32),
+                        Parameter(id="unk", start="$id.stop", length=4, type=ctype.int32),
```

Comment:
is this `unk` used somewhere?

Reply:
Nope, just added it here as it made in l2j to be sure how many bits are comes.
According to my observation, unk always the same.

Reply:
They need to define it because of the way they handle data.
They read it by chunks, we read it all at once when receiving it from asyncio protocol.
We don't need to read it

Reply:
got it, I will remove this in next commit so.